EST.MIT.InvoiceImporter.Function/Configuration/ServiceConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Azure.Storage.Queues;
+using EST.MIT.InvoiceImporter.Function.Interfaces;
+using EST.MIT.InvoiceImporter.Function.Services;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Win32;
+
+namespace EST.MIT.InvoiceImporter.Function.Configuration
+{
```

Comment:
Please update to a file scoped namespace for consistency.

Reply:
Resolved